benchmarks/src/test/java/org/apache/druid/benchmark/BenchmarkDataGeneratorTest.java

```diff
@@ -431,19 +431,19 @@ public void addRow(InputRow row)
     public void printStuff()
     {
       System.out.println();
-      for (String dim : dimensionMap.keySet()) {
+      for (Map<Object, Integer> dim : dimensionMap.values()) {
         System.out.println("DIMENSION " + dim + "\n============");
-        Map<Object, Integer> valueMap = dimensionMap.get(dim);
+        Map<Object, Integer> valueMap = dim;
```

Comment:
Unnecessary variable, the iteration variable should be called `valueMap` itself.